pkg/terminal/command.go

```diff
@@ -2353,13 +2353,17 @@ func PrintStack(formatPath func(string) string, out io.Writer, stack []api.Stack
 	d := digits(len(stack) - 1)
 	fmtstr := "%s%" + strconv.Itoa(d) + "d  0x%016x in %s\n"
 	s := ind + strings.Repeat(" ", d+2+len(ind))
+	printed := 0
 
 	for i := range stack {
+		if !filter(stack[i]) {
```

Comment:
Are we filtering in or out? I would suggest names like include/exclude to make this clearer.